partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -48,11 +48,11 @@
       "The schema generator to use with partitioners.";
   public static final String SCHEMA_GENERATOR_CLASS_DISPLAY = "Schema Generator Class";
 
-  public static final String PARTITION_FIELD_NAME_CONFIG = "partition.field.name";
+  public static final String PARTITION_FIELD_NAME_CONFIG = "partition.field.names";
   public static final String PARTITION_FIELD_NAME_DOC =
-      "The name of the partitioning field when FieldPartitioner is used.";
```

Comment:
Deprecation of a config property can be flagged only at the documentation level for the moment. For instance, we should extend the old config doc in a way that notifies of the new config and the deprecation of the old one: 

Example:
```java 
public static final String PARTITION_FIELD_NAME_DOC = 
    "The name of the partitioning field when FieldPartitioner is used"
    + " (This property is deprecated. Use " + PARTITION_FIELD_NAMES_CONFIG + " instead).";
```
(something like that).

Reply:
And add `PARTITION_FIELD_NAMES_CONFIG` (plural) of course.

Reply:
connect UI in CC only supports adding a single field now  . Wouldn't that need changes or  can we pass a  comma separated strings from CC (struct1.fieldname,struct2.fieldname) ?